esmvaltool/cmorizers/obs/cmor_config/ERA-Interim-Land.yml

```diff
@@ -0,0 +1,19 @@
+---
+# Global attributes of NetCDF file
+attributes:
+  dataset_id: ERA-Interim-Land
+  project_id: OBS6
+  tier: 3
+  version: '1'
+  modeling_realm: reanaly
+  source: 'https://apps.ecmwf.int/datasets/data/interim-land/type=an/'
+  reference: 'era-interim-land'
+  comment: |
+    'Contains modified Copernicus Climate Change Service Information {year}'
+
+# Variables to CMORize
+variables:
+  sm:
+    mip: E6hr
+    raw: swvl1
+    file: 'eraintland_swvl1_????.nc'
```

Comment:
```suggestion
    file: 'ERA-Interim-Land_swvl1_monthly_????.nc'
```

Reply:
I don't think we need to distinguish between `ERA-Interim` and `ERA-Interim-Land`: since the variable names are unique there is no risk of confusion.

We should also use a single dataset name (`ERA-Interim`) in our observational data-pool.

Reply:
> since the variable names are unique there is no risk of confusion.

I think this is not correct, e.g. for soil moisture, both `ERA-Interim` and `ERA-Interim-Land` have the variable `swvl1`.

Reply:
Are they actually different?
That would be surprising, since the [parameter database](https://apps.ecmwf.int/codes/grib/param-db/) does not differentiate by domain.

Reply:
Yes, they **are** actually different, these are two different datasets. The `land` version (this holds for `ERA-Interim` as well as for `ERA5`) is created by running a specific land surface model with atmospheric variables from the `normal` reanalysis as input.

Reply:
Ok, thank you!